keps/sig-api-machinery/20190226-network-proxy.md

```diff
@@ -0,0 +1,577 @@
+---
+title: Network Proxy
```

Comment:
Would it be possible to rename this KEP to something like "API Server Network Proxy" or "Control Network Proxy" so it's not confused with kube-proxy?

Reply:
+10

Reply:
There are only two hard things in Computer Science: cache invalidation, naming things and off by one errors.

Can do.